Add recognition of 2i stats

martinsumner opened the pull request:
Test for new log in index queries, and updates to riak_stats.

martinsumner committed:
Add recognition of 2i stats

martinsumner committed:
Add new windex stats to test

martinsumner committed:
Update verify_riak_stats.erl

martinsumner committed:
Check new stats via statsfold test

Also extends basic_upgrade test to test a slightly less trivial volume of data, and include in a group.

martinsumner committed:
Ensure queue by repeating stats fetch, and reducing wait

ThomasArts commented on tests/verify_basic_upgrade.erl:
```diff
@@ -22,15 +22,41 @@
     OldVsn = proplists:get_value(upgrade_version, TestMetaData, previous),
 
-    Nodes = [Node1|_] = rt:build_cluster([OldVsn, OldVsn, OldVsn, OldVsn]),
+    [Nodes] = 
```
This looks different to the eye.
Is there an extra list around result of `rebuild_cluster` now..?

ThomasArts replied:
Ok, it's `rebuild_clusters` with an `s`.

martinsumner committed:
Intermittent failure - with immediate delete

This test would intermittently fail.  The failure is due to immediate delete not always reaping tombstones (perhaps due to race conditions).  This appears to be happening to random keys.

We now tolerate some failure for immediate delete to reap tombstones, rather than failing the test.  Testing the function of immediate delete is not the purpose of this test.
